README.md

```diff
@@ -35,6 +35,12 @@ the review process quicker and more streamlined.
 
 ## Install
 
+To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
+The oldest Node.js version we support is 4.2.0.
```

Comment:
What happened before 4.2.0 (and > 0.12) so that we don't support them?
I think that in the future, we could get away with supporting current stable and LTS proposed by the Node.js website, what do you think?